libs/@local/hash-isomorphic-utils/src/generate-system-types.ts

```diff
@@ -0,0 +1,40 @@
+// We need to import this file to load the environment variables –– ontology-types.ts relies on SYSTEM_USER_SHORTNAME.
+// This script is used in the backend only, but the outputs are used in the frontend, thus the following eslint-disable.
+// eslint-disable-next-line no-restricted-imports
+import "@local/hash-backend-utils/environment";
+
+import { codegen, CodegenParameters } from "@blockprotocol/graph/codegen";
+import slugify from "slugify";
+
+import { types } from "./ontology-types";
```

Comment:
Just want to check you're aware we're not generating the types for the linear entity types here yet, is that something we're planning to do?

Reply:
I forgot these were separate – added in https://github.com/hashintel/hash/pull/2852/commits/eb32f8c6e2e7ed6f76d88520d1b1fd58e5d0edf3

These have to be generated in a separate step, because if we try and do it all at once we'll get clashes between Linear `User` and the system `User`, and they end up with `1`s added to them, which seems worse than just putting them in separate files.